silent/tests/testdata/su-err-pr-exists-security.txt

```diff
@@ -0,0 +1,41 @@
+! dependabot update -f input.yml --local . --updater-image ghcr.io/dependabot/dependabot-updater-silent
+stderr pull_request_exists_for_security_update
+stdout '{"data":{"error-type":"pull_request_exists_for_security_update","error-details":{"updated-dependencies":\[{"dependency-name":"dependency-a","dependency-version":"1.2.4"}\]}},"type":"record_update_job_error"}'
+
+# An existing pull request exists for 1.2.4, which is the security version required, but not the latest.
+
+-- manifest.json --
+{
+  "dependency-a": { "version": "1.2.3" }
+}
+
+-- dependency-a --
+{
+  "versions": [
+    "1.2.3",
+    "1.2.4",
+    "1.2.5"
+  ]
+}
+
+-- input.yml --
+job:
+  package-manager: "silent"
+  dependencies:
+    - dependency-a
+  source:
+    directory: "/"
+    provider: example
+    hostname: example.com
+    api-endpoint: https://example.com/api/v3
+    repo: dependabot/smoke-tests
+  security-advisories:
+    - dependency-name: dependency-a
+      affected-versions:
+        - <= 1.2.3
+      patched-versions: []
```

Comment:
I don't think it's material to the test working, so feel free to ignore: I think realistically `patched-versions` would include `1.2.4`.

Reply:
I've never seen that field set to a value!

Reply:
Interesting; do you know why that is the case?

Reply:
I don't, would need to dig into that.